test/pricer_rule/utils.js

```diff
@@ -115,5 +125,6 @@ module.exports.createTokenEconomy = async (accountProvider) => {
     quoteCurrencyCode,
     priceOracleInitialPrice,
     priceOracle,
+    eip20TokenDecimals,
```

Comment:
If we need this exposed, please, place it close to `token` and let's name it tokenDecimals for consistency.

Reply:
Done!